extensions/sql-database-projects/src/sqldbproj.d.ts

```diff
@@ -323,7 +323,8 @@ declare module 'sqldbproj' {
 		sqlDW = 'Azure Synapse SQL Pool',
 		sqlEdge = 'Azure SQL Edge',
 		sqlDwServerless = 'Azure Synapse Serverless SQL Pool',
-		sqlDwUnified = 'Synapse Data Warehouse in Microsoft Fabric'
+		sqlDwUnified = 'Synapse Data Warehouse in Microsoft Fabric',
+		sqlDbFabric = 'Fabric mirrored SQL database (preview)'
```

Comment:
nit: start all the words with capitals
```suggestion
		sqlDbFabric = 'Fabric Mirrored SQL Database (Preview)'
```